packages/iris-grid/src/IrisGrid.tsx

```diff
@@ -3885,8 +3899,10 @@ export class IrisGrid extends Component<IrisGridProps, IrisGridState> {
               model={model}
               movedColumns={movedColumns}
               userColumnWidths={userColumnWidths}
+              columnHeaderGroups={model.getColumnHeaderGroups()}
```

Comment:
Hmmm why are you using the `model.getColumnHeaderGroups()` instead of the `columnHeaderGroups` from state?